jobserver/templates/workspace_detail.html

```diff
@@ -78,21 +78,19 @@ <h3>{{ workspace.name }}</h3>
 
       <div class="d-flex">
         {% if can_use_releases %}
-        {% if user_can_view_outputs %}
-        <a class="btn btn-primary mr-2" href="{{ workspace.get_current_outputs_url }}">
-          Current Outputs
-        </a>
-        <a class="btn btn-primary mr-2" href="{{ workspace.get_outputs_url }}">Outputs</a>
-        <a class="btn btn-primary mr-2" href="{{ workspace.get_current_outputs_url }}">
-          Publish
-        </a>
-        {% endif %}
 
         {% if user_can_view_files %}
-        <a class="btn btn-primary mr-2" href="{{ workspace.get_files_url }}">Backend Files</a>
+        <a class="btn btn-primary mr-2" href="{{ workspace.get_files_url }}">Current Outputs</a>
```

Comment:
I wonder if "Level 4 Outputs" is less ambiguous, as "Current" could be interpreted as current level 4, current released, currently published?